code/modules/mob/living/combat/parry.dm

```diff
@@ -303,6 +303,27 @@
 				src.visible_message(span_boldwarning("<b>[src]</b> ripostes [user] with [W]!"))
 			else
 				src.visible_message(span_boldwarning("<b>[src]</b> parries [user] with [W]!"))
+			// Special: jitterskull heavily degrades parrying items instead of being hard-countered by parries
```

Comment:
This is probably bad practice codewise, throwing up an edgecase into the parry code specifically for one mob.
If you want the simple mob do better against people parrying, give the mob unarmed skill. See summoned mage mobs for an example.